public/index.html

```diff
@@ -21,6 +21,7 @@
     <title></title>
     <meta name="keywords" content="Personal Assistant Free Software Artificial Intelligence Chat Bot Machine Learning Scientific User Support IoT"/>
     <meta name="description" content="Chat with an Open Source Personal Assistant"/>
+    <meta name="theme-color" content="#4285f4">
```

Comment:
Please change only those files related to issue

Reply:
That was included in previous PR. I've removed that!